conf/nginx.conf

```diff
@@ -1,33 +1,18 @@
-#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
-location __PATH__/ {
```

Comment:
Are you sure about that?

Reply:
yes, I think the app won't be able to install in a sub_path...
The upstream app is in early development
I haven't checked all, I'm currently debugging the installation. 
It installs but I have "blank page", I think the app is not ready for production

Reply:
Ok, it was to be sure that it is intended